numpy/testing/tests/test_utils.py

```diff
@@ -108,11 +108,13 @@ def test_0_ndim_array(self):
         x = np.array(473963742225900817127911193656584771)
         y = np.array(18535119325151578301457182298393896)
 
-        with pytest.raises(AssertionError) as exc_info:
+        expected_msg = ('Mismatched elements: 1 / 1 (100%)\n'
+                        'Max absolute difference among violations: '
+                        '455428622900749238826454011358190875\n'
+                        'Max relative difference among violations: '
+                        '24.571119015281806\n')
```

Comment:
The tortured grammar around plurals here sort of make me want to special-case this message for scalars. Is that painful to do here?